src/ui/public/vislib/visualizations/point_series/_point_series_chart.js

```diff
@@ -1,20 +1,30 @@
-import d3 from 'd3';
 import _ from 'lodash';
-import ChartProvider from 'ui/vislib/visualizations/_chart';
-import TooltipProvider from 'ui/vislib/components/tooltip';
 import errors from 'ui/errors';
 
 export default function PointSeriesChartProvider(Private) {
 
-  const Chart = Private(ChartProvider);
-  const Tooltip = Private(TooltipProvider);
-  const touchdownTmpl = _.template(require('ui/vislib/partials/touchdown.tmpl.html'));
+  class PointSeriesChart {
```

Comment:
It seems like this has evolved away from `XYZChart`, should it just be `PointSeri`? `BasePointSeri`? `AbstractPointSeri`?